replit_river/session.py

```diff
@@ -145,12 +150,7 @@ async def serve(self) -> None:
                 try:
                     await self._handle_messages_from_ws(tg)
                 except ConnectionClosed:
-                    if self._retry_connection_callback:
-                        self._task_manager.create_task(
-                            self._retry_connection_callback()
-                        )
-
-                    await self._begin_close_session_countdown()
+                    await self._handle_connection_closed()
```

Comment:
seems like we lost the _retry_connection_callback callback here?

Reply:
That is already handled down inside `close_websocket`: https://github.com/replit/river-python/blob/235406c66b60a8d446ab429d54cbe60a34cc071f/replit_river/session.py#L454-L455